src/amo/components/SearchSort/index.js

```diff
@@ -0,0 +1,79 @@
+import classNames from 'classnames';
+import React, { PropTypes } from 'react';
+import { compose } from 'redux';
+
+import SearchSortLink from 'amo/components/SearchSort/SearchSortLink';
+import translate from 'core/i18n/translate';
+
+import './SearchSort.scss';
+
+
+const DEFAULT_SORT = 'relevance';
+
+export class SearchSortBase extends React.Component {
+  static propTypes = {
+    filters: PropTypes.object.isRequired,
+    i18n: PropTypes.object.isRequired,
+    pathname: PropTypes.string.isRequired,
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = { sortVisible: false };
+  }
+
+  onClick = (event) => {
+    event.preventDefault();
+    this.toggleSort();
+  }
+
+  sortOptions() {
+    const { i18n } = this.props;
+
+    return [
+      { sort: 'updated', text: i18n.gettext('Recently Updated') },
+      { sort: 'relevance', text: i18n.gettext('Relevance') },
+      { sort: 'users', text: i18n.gettext('Most Users') },
+      { sort: 'rating', text: i18n.gettext('Top Rated') },
+    ];
+  }
+
+  toggleSort() {
+    // Blur the search sort toggle if we're closing it.
+    if (this.state.sortVisible) {
+      this.searchToggle.blur();
```

Comment:
The `onClick` (and by extension taps on mobile) now passes an argument to blur the sort when closed. Keyboard focus is not changed when closed:

![jan-14-2017 18-29-15](https://cloud.githubusercontent.com/assets/90871/21957097/8245c8b6-da87-11e6-8a0d-d132bfa3c121.gif)

Good call 👍